src/components/Login.jsx

```diff
@@ -0,0 +1,58 @@
+import { GoogleAuthProvider, signInWithPopup } from "firebase/auth";
+import { useQuery, useMutation } from "@tanstack/react-query";
+import { useNavigate } from "react-router-dom";
+import { firebaseAuth } from "./app/firebaseAuth";
+import logo from "../assets/dataface_logo.png";
+import googleLogo from "../assets/google_logo.png";
+import fetchData from "../utils/axios";
```

Comment:
```suggestion
import { GoogleAuthProvider, signInWithPopup } from "firebase/auth";
import { useQuery, useMutation } from "@tanstack/react-query";
import { useNavigate } from "react-router-dom";

import { firebaseAuth } from "./app/firebaseAuth";
import fetchData from "../utils/axios";

import logo from "../assets/dataface_logo.png";
import googleLogo from "../assets/google_logo.png";
```

하나의 컴포넌트에서 import 해오는 것들이 많을때 저는 약간 경로 혹은 관심사에 따라 개행을 통해 가독성을 높이고 분리를 해주는 편입니다! 
일단은 라이브러리를 import 했는지, 경로에서 import 했는지에 따라 나눠 봤는데 firebase 따로 이미지 따로 함수 따로 나눠도 괜챃을것 같아요
괜찮으시다면 import 종류에 따라 개행으로 분리 부탁드리겠습니다~~!

Reply:
네 알겠습니다!